exercises/scrabble-score/lib/scrabble_score.dart

```diff
@@ -0,0 +1,3 @@
+class ScrabbleScore {
+  // your code here
```

Comment:
Change to `// Put your code here`

Add a line after the `}`

Reply:
Added

Reply:
Sorry. Added `Put`